apps/backend/src/main.ts

```diff
@@ -1,9 +1,32 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
+import { swaggerDocumentOptions } from './swagger/swagger.config';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
+  const swaggerSettings = new DocumentBuilder()
```

Comment:
In the file under the `frameworks` folder, create a class to abstract the whole necessary logic to start Swagger. Then you call it here

Reply:
Done!